src/api/app/models/bs_request_action_tab_visibility.rb

```diff
@@ -0,0 +1,20 @@
+class BsRequestActionTabVisibility < ApplicationComponent
```

Comment:
This is not a ViewComponent is it?

Reply:
Is this a [Decorator](https://thoughtbot.com/blog/evaluating-alternative-decorator-implementations-in#the-quotplain-old-ruby-objectquot-poro-decorator)?